content/en/docs/kitex/FAQ/_index.md

```diff
@@ -33,10 +33,15 @@ description: >
 
 ## Kitex Code Generation Tool
 
-**Q:'not enough arguments' problem when installing the code generation tool**  
+**Q:'not enough arguments' problem when installing the code generation tool**
 * Please try: "go mod：GO111MODULE=on go get github.com/cloudwego/kitex/tool/cmd/kitex@latest"
 
-**Q: Would code generated by new interface overwrite handler.go**  
+**Q: Would code generated by new interface overwrite handler.go**
 * Generated code under kitex_gen/ will be overwritten. But handler.go of server will not be overwritten, only new methods will be added correspondingly.
 
+**Q: Does the template in the current code generation tool support user customization?**
+* At present, there is no plan to support customized templates, because the parameter transfer design will be much more complex. The current plugin mechanism can completely realize any equivalent function
+
+**Q: Whether the "–type" in the code generation tool can be automatically determined by the suffix of the IDL file name and "–module" is changed to default through the module information in go.mod is determined, and use "-module" parameter when multiple files ?**
+* "-type" will be studied, because suffixes are limited now, but it is hard to say whether any suffixes will be supported in the future -module this is because when the internal version of kitex is released, go module has not been widely used, so the default behavior is based on "GOPATH/src", which has been retained for compatibility reasons. Let's see if we can adjust the open source version.
```

Comment:
* On "-type" we need further discussion. Though filename suffix are currently restricted to a few certain types, arbitrary value may be supported in the future.
* We'll consider the advice for "-module".